server/config/routes.js

```diff
@@ -27,45 +30,49 @@ module.exports = function(app) {
   app.post('/api/users/signin', controller.auth.authenticateUser);
 
   
-  //log out
+  //log out???
     //destroy session
     //route user to sign in page
   app.get('/api/users/logout', controller.auth.logout);
   
-  //get one story for a given user
+  //get one story for a given user (COMPLETED V1/CHECKED)
```

Comment:
These comments would be more useful in front of the actual code they are describing.